recipes/libsemigroups_cppyy/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "libsemigroups_cppyy" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "a3c194f4bbb70df2abf64d3133115378931581627bfb32bd3e78c8ee4f47d1f5" %}
+{% set repo = "https://github.com/libsemigroups/libsemigroups_cppyy" %}
+{% set home = "https://libsemigroups.github.io/libsemigroups/" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: {{ repo }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - libsemigroups >=1.0.7
+    - cppyy >=1.6.2
+    - cppyy-backend >=1.10.8
+    - cppyy-cling >=6.18.2.7
+    - CPyCppyy >=1.10.2
```

```suggestion
    - cppyy >=1.6.2
```

`cppyy` depends on these packages. I guess `libsemigroups_cppyy` is not using any of the others explicitly so imho they should not be mentioned.

---

These packaged are still in the recipe. Was this intentional?

---

Ah, no unintentional I removed them now.